modules/heat_conduction/include/kernels/JouleHeatingHeatGeneratedAux.h

```diff
@@ -0,0 +1,35 @@
+//* This file is part of the MOOSE framework
```

Comment:
Create an "auxkernels" folder under include and move this file into that folder.